src/wp-admin/options-general.php

```diff
@@ -506,6 +506,34 @@
 ?>
 </select></td>
 </tr>
+
+
+<tr>
+<th scope="row"><label for="blocks_compatibility_level"><?php _e( 'Blocks compatibility' ); ?></label></th>
```

Comment:
For consistency with other headings, "compatibility" should be capitalized here "Blocks Compatibility".